admin_guide/cluster_management.rst

```diff
@@ -28,12 +28,62 @@ To add a new node to the cluster, you first need to add the DNS name or IP addre
 
    SELECT * from master_add_node('node-name', 5432);
 
-In addition to the above, if you want to move existing shards to the newly added worker, Citus Enterprise provides an additional rebalance_table_shards function to make this easier. This function will move the shards of the given table to make them evenly distributed among the workers.
+The new node is available for shards of new distributed tables. Existing shards will stay where they are unless redistributed, so adding a new worker may not help performance without further steps.
+
+Rebalancing Shards
+------------------
+
+If you want to move existing shards to a newly added worker, Citus Enterprise and Citus Cloud provide a :ref:`rebalance_table_shards` function to make it easier. This function will move the shards of a given table to make them evenly distributed among the workers.
 
 ::
 
-	select rebalance_table_shards('github_events');
+  SELECT rebalance_table_shards('github_events');
+
+Shard rebalancing is configurable to act by either of two methods:
+
+1. Logical replication or
+2. COPY with blocked writes
+
+The first option is preferred because it is fully "online," meaning applications communicating with the database do not experience downtime for reads or writes, except a very brief block on writes at the end of replication during a metadata update. The second option is online for reads only; writes are blocked while copying data from source to destination shard, which can take time.
```

Comment:
Any chance we can make "brief block on writes" an actual ballpark number? (i.e. "seconds"?) cc @sumedhpathak

Reply:
@onderkalaci's pseudo code for the new rebalancing is:

1. Create the shard on the target node
1. Setup the logical replication for the shard from the source node to the target node
1. Wait until the shard on the target catches up with the source
1. Block modifications to the shard
1. Wait until the shard on the target catches up with the source after we’ve blocked the modifications
1. Drop the logical replication related setup
1. Drop the source shard
1. Update the metadata to reflect the shard move 
1. Unblock writes to the shard (the awaiting modifications now sees the new metadata)
1. Continue with the next shard

So we're blocking during operations 5-8, however long that takes.

Reply:
@begriffs I see - so might be a bit difficult to get a good measurement there, since I assume Step 5 in particular would have variable runtime depending on replication performance.

Maybe leave as is for now, I could see us getting better ballpark numbers on this over time, as we see customers using it.